src/react-jw-player.jsx

```diff
@@ -43,6 +43,11 @@ class ReactJWPlayer extends Component {
       existingScript.onload = getCurriedOnLoad(existingScript, this._initialize);
     }
   }
+  componentDidUpdate(prevProps) {
+    if (prevProps.playlist !== this.props.playlist || prevProps.file !== this.props.file) {
```

Comment:
This is great, but I'd like to use `shouldComponentUpdate` for the logic determining whether we should initialize or not, and let `componentDidUpdate` just call initialize. Wdyt?